openshift/template.yaml

```diff
@@ -0,0 +1,125 @@
+apiVersion: v1
+kind: Template
+labels:
+  template: 3scale-admin-gateway
+metadata:
+  name: 3scale-admin-gateway
+  annotations:
+    description: 3scale-admin-gateway
+objects:
+- apiVersion: v1
+  kind: DeploymentConfig
+  metadata:
+    labels:
+      service: 3scale-admin-gateway
+    name: 3scale-admin-gateway
```

Comment:
We try to prefix all new f8a services with `f8a-` prefix. It's not mandatory though.